### ISSUE TITLE
[Bug Report] Tabs display content from all tab on page load

### ISSUE THREAD

**Issue by aldarund** [vuetifyjs/vuetify#4397]
### Versions and Environment
**Vuetify:** 1.1.0-rc.2
**Vue:** 2.5.16
**Browsers:** Chrome 68.0.3440.33
**OS:** Windows 10

### Steps to reproduce
Open codepen. Or first example from https://next.vuetifyjs.com/en/components/tabs

### Expected Behavior
Only first tab content shown

### Actual Behavior
Content from all tabs shown

### Reproduction Link
[https://codepen.io/aldarund/pen/wXxbNz?editors=1010](https://codepen.io/aldarund/pen/wXxbNz?editors=1010)

**Comment by Alexays** [comment]
I've noticed this problem too just add an id to tabs to fix it.

**Comment by jamescoffman23** [comment]
I had to change my "activeTab" v-model to numeric instead of strings and it worked for me.

**Comment by KaelWD** [comment]
Yeah this is because of #3641, it needs to still work with strings too.

**Comment by kauaicreative** [comment]
simple adding, id to the 1st v-tab-item fixes the issue for me.
`<v-tab-item id v-for="n in 3"  @:key="n">`
